README.md

```diff
@@ -306,6 +308,19 @@ When the `bandwidth` property is set (bits per second), it will be used in
 the calculation for initial playlist selection, before more bandwidth
 information is seen by the player.
 
+##### storeBandwidthInLocalStorage
```

Comment:
Is this option needed? Any reason to not just have storage always enabled, and allow the setting of whether its actually used?